MobileCenterCrashes/MobileCenterCrashes/Internals/MSCrashesInternal.h

```diff
@@ -23,4 +23,10 @@
  */
 - (void)configureCrashReporterWithUncaughtExceptionHandlerEnabled:(BOOL)enableUncaughtExceptionHandler;
 
+/*
+ * Track handled exception directly as model form.
+ * This API is not public and is used by wrapper SDKs.
+ */
++ (void)trackModelException:(MSException *)exception;
```

Comment:
Any intention that you don't use same method name as Android?

Reply:
I thought objective c would not allow later to have  `trackException:(NSException *)` when we want to support that. Can you confirm the name would collide?

Reply:
@TroubleMakerBen @clpolet?

Reply:
NSExceptions are fatal in objC. If we have "unhandled error tracking" (or whatever you call it), iOS will have something like `trackError:(NSError *)` or `trackError:(MSError*)`.

Reply:
@TroubleMakerBen this clearly shows NSException can be caught: https://stackoverflow.com/questions/35119531/catch-objective-c-exception-in-swift

Reply:
OK. So this is only used by Xamarin.iOS and you want to prevent from having a same method name when we support trackException API.

Reply:
I am not aware of anything preventing is from using `trackException`.

Reply:
Except if there will be another `trackException:` method later on the same class with a different parameter/return value.

Reply:
@clpolet yes that's exactly the point. Avoid collision with a future trackException(NSException *) if that ever becomes a thing.